core/blockchain.go

```diff
@@ -1341,6 +1398,9 @@ func (bc *BlockChain) writeBlockWithState(block *types.Block, receipts []*types.
 	if err != nil {
 		return err
 	}
+	if bc.triedb.Scheme() == rawdb.PathScheme {
```

Comment:
Perhaps add a comment why we're not doing anything else in path mode. I guess because there's no GC going on?